DQM/EcalMonitorTasks/interface/TrigPrimTask.h

```diff
@@ -49,8 +49,10 @@ namespace ecaldqm {
     /*     bool HLTCaloBit_; */
     /*     bool HLTMuonBit_; */
 
-    std::array<int, nBXBins + 1> bxBinEdges_;
+    std::vector<int> bxBinEdges_;
+    std::vector<int> bxBinEdgesFine_;
     double bxBin_;
+    double bxBinFine_;
```

Comment:
Hi @alejands and @jfernan2 , could you please remind me the justification for using _double_ here ? thanks

Reply:
We use `bxBinFine_` as the `x` with which to fill a histogram. It is assigned a half-integer value here to avoid ambiguity https://github.com/cms-sw/cmssw/blob/master/DQM/EcalMonitorTasks/src/TrigPrimTask.cc#L95. Our fill function takes in a double for this value, so it will be type cast into a double at runtime anyway.

Reply:
OK , thanks @alejands for the clarification.